### PR TITLE
Check err in zilliqa model

### LINKED ISSUE #1042: [Observer-Zilliqa] panic: runtime error: invalid memory address or nil pointer dereference
```shell
panic: runtime error: invalid memory address or nil pointer dereference
4/21/2020 6:18:04 AM [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x59d7a8]
4/21/2020 6:18:04 AM
4/21/2020 6:18:04 AM goroutine 2592 [running]:
4/21/2020 6:18:04 AM math/big.(*Int).Mul(0xc00024d718, 0x0, 0x0, 0xa)
4/21/2020 6:18:04 AM /usr/local/go/src/math/big/int.go:164 +0x48
4/21/2020 6:18:04 AM github.com/trustwallet/blockatlas/platform/zilliqa.(*TxRPC).toTx(0xc00046e0c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
4/21/2020 6:18:04 AM /build/platform/zilliqa/model.go:74 +0x205
4/21/2020 6:18:04 AM github.com/trustwallet/blockatlas/platform/zilliqa.(*RpcClient).GetTxInBlock(0xc0001e02a8, 0x81072, 0x0, 0x0, 0xc00011dab8, 0x4190ae, 0x7f0433645fe8)
4/21/2020 6:18:04 AM /build/platform/zilliqa/rpc.go:69 +0x3f2
4/21/2020 6:18:04 AM github.com/trustwallet/blockatlas/platform/zilliqa.(*Platform).GetBlockByNumber(0xc0001e0280, 0x81072, 0x44032c, 0xc0000a83b8, 0x2)
4/21/2020 6:18:04 AM /build/platform/zilliqa/block.go:24 +0x56
4/21/2020 6:18:04 AM github.com/trustwallet/blockatlas/services/observer/parser.getBlockByNumberWithRetry(0x5, 0x12a05f200, 0xc00024dea0, 0x81072, 0xc000355b00, 0x0, 0xc000637680)
4/21/2020 6:18:04 AM /build/services/observer/parser/parser.go:267 +0x3a
4/21/2020 6:18:04 AM github.com/trustwallet/blockatlas/services/observer/parser.fetchBlock(0x7f043132cb70, 0xc0001e0280, 0x81072, 0xc0001a27e0, 0xc0003d8fa8, 0x720a06)
4/21/2020 6:18:04 AM /build/services/observer/parser/parser.go:160 +0x8d
4/21/2020 6:18:04 AM github.com/trustwallet/blockatlas/services/observer/parser.FetchBlocks.func1(0x10825c0, 0xc0002a21c0, 0x7f043132cb70, 0xc0001e0280, 0xdf4385, 0xf, 0x6fc23ac00, 0xf4240, 0xa, 0x3c, ...)
4/21/2020 6:18:04 AM /build/services/observer/parser/parser.go:127 +0x90
4/21/2020 6:18:04 AM created by github.com/trustwallet/blockatlas/services/observer/parser.FetchBlocks
4/21/2020 6:18:04 AM /build/services/observer/parser/parser.go:125 +0x1b3
```

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #1042, Changes:

1. run `go mod tidy`
2. return nil if error occurs in `zilliqa/model.go`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,29 +3,40 @@ module github.com/trustwallet/blockatlas
 go 1.14
 
 require (
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
```
Comment (EnoRage) [inline go.mod:6]: We need to remove / indirect deps

Comment (hewigovens) [reply go.mod]: I reverted, can you check `go mod tidy` then?

Comment (EnoRage) [reply go.mod]: https://github.com/golang/go/issues/27428#issuecomment-417934130

Comment (hewigovens) [reply go.mod]: from the comment, we should include //indirect? always produce stable build

Comment (EnoRage) [reply go.mod]: If compiler cannot build it. We use Dockerfile for builds. There is no reason to add something if it works without it. Just my position about it, not a fact of course

Comment (hewigovens) [reply go.mod]: I don’t use docker when I develop

Comment (EnoRage) [reply go.mod]: Sure. But dev env != prod env

**Comment by vikmeup** [comment]
👍

**Pull request merged**